<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dupfind</class>
 <widget class="QMainWindow" name="dupfind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DupFind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_settings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>81</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_pick_Dir">
             <property name="text">
              <string>Directory to scan:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_dir"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_browse">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_scan">
             <property name="text">
              <string>Start Scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_results">
        <property name="title">
         <string>Click an item to get a list of found duplicates</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeWidget" name="treeWidget_dup">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Size</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Modified</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Created</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Accessed</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Duplicates</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QListWidget" name="listWidget_all"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>dupfind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
